### PR TITLE
Extend trending words signal

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Related PRs
This PR is based on #1270 

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: docs/user-triggers-api.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   - [Trigger structure](#trigger-structure)
   - [Settings fields](#settings-fields)
   - [Examples](#examples)
-    - [Example settings structure for `price_absolute_change`](#example-settings-structure-for-priceabsolutechange)
-    - [Example settings structure for `price_percent_change`](#example-settings-structure-for-pricepercentchange)
-    - [Example settings structure for `daily_active_addresses`](#example-settings-structure-for-dailyactiveaddresses)
-    - [Example settings structure for `trending_words`](#example-settings-structure-for-trendingwords)
-    - [Example settings structure for `price_volume_difference`](#example-settings-structure-for-pricevolumedifference)
-    - [Example settings structure for `eth_wallet`](#example-settings-structure-for-ethwallet)
+    - [Example settings structure for `price_absolute_change`](#example-settings-structure-for-price_absolut_echange)
```
Comment (kalkov) [inline docs/user-triggers-api.md]: absolut -> absolute :cocktail:

File: lib/sanbase/signals/trigger/settings/trending_words_trigger_settings.ex [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-  Trigger settings for daily trending words signal.
-  The signal is sent at the configured `trigger_time` and sends the last set of
-  trending words that was calculated.
-  Currenly the trending words are calculated 3 times per day - at 01:00, 08:00
-  and 14:00 UTC time
+  Trigger settings for trending words signal.
+
+  The signal supports the following operations:
+
+  1. Send at the list of trending words at predefined time every day
```
Comment (tspenov) [inline lib/sanbase/signals/trigger/settings/trending_words_trigger_settings.ex]:
```suggestion
  1. Send the list of trending words at predefined time every day
```

File: lib/sanbase/signals/trigger/settings/trending_words_trigger_settings.ex [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+  2. Send a signal if some word enters the list of trending words.
+  3. Send a signal if some project enters the list of trending words
```
Comment (tspenov) [inline lib/sanbase/signals/trigger/settings/trending_words_trigger_settings.ex]: Maybe also add `4. Send a signal if any of the projects in a watchlist enters the list of trending words` ?

Comment (tspenov) [reply lib/sanbase/signals/trigger/settings/trending_words_trigger_settings.ex]: Or maybe watchlist here is reduced to a list of projects so this module doesn't know if it is a watchlist?

Comment (IvanIvanoff) [reply lib/sanbase/signals/trigger/settings/trending_words_trigger_settings.ex]: It doesn't know if it's a watchlist but we need document it as the translation is an implementation detail

**Pull request merged**